### ISSUE TITLE
Pickle files

### ISSUE THREAD

**Issue by wlcosta** [hassony2/obman_train#4]
Hi,
I'm trying to replicate the results by training on the provided datasets. A few pickle files are missing from the repository:

- For **ShapeNetCore.V2** the training script looks for a file called `surface_points.pkl`, which is not present in the dataset downloaded. By looking at the code I have found a line of the script which replaces `model_normalized.pkl` to `surface_points.pkl`:
https://github.com/hassony2/obman_train/blob/d90781e515527358577b10d76011cab56d2030c6/handobjectdatasets/obman.py#L455-L456
but on the ShapeNet folders, none of the normalized models are pickled. Maybe if I load the `*.json` files and pickle them that would be ok?

- For the part related to the hand-only model, there are also a few pickle files that are not present in the repository:
https://github.com/hassony2/obman_train/blob/cd86a845b0e17f520d6415cee3705cf24e489d05/handobjectdatasets/stereohands.py#L84-L90
https://github.com/hassony2/obman_train/blob/cd86a845b0e17f520d6415cee3705cf24e489d05/handobjectdatasets/fhbhands.py#L123-L129
The structure of the code suggests that these files are in a separated location. Did you mean to share these files? If yes, could you share them?

Finally, I want to congratulate you for the awesome work. Cheers!

**Comment by hassony2** [comment]
Hi @wlcosta,

Thank you for your interest in our work and your spot-on remarks !

About the missing pickle files: these are actually not used in the final versions, so I just discarded these lines and pushed the new loading files. At some point I experimented with normalizing bone lengths to a given skeleton.

For the preprocessing files:
- `model_normalized.pkl` if created by [shapenet_cache.py](https://github.com/hassony2/obman/blob/master/shapenet_cache.py)
- `surface_points.pkl` files are created by [shapenet_samplepoints.py](https://github.com/hassony2/obman/blob/master/shapenet_samplepoints.py)

The training code received less attention during cleaning compared to the demo one :) If you run into any other issues don't hesitate to reach out ! 

Best,

Yana

**Comment by Ivamcoder** [comment]
Hi @wlcosta , do you know that ?[https://github.com/hassony2/obman_train/issues/7](url)
